public/apps/login/login-page.tsx

```diff
@@ -110,6 +105,7 @@ export function LoginPage(props: LoginPageDeps) {
       <EuiForm component="form">
         <EuiFormRow>
           <EuiFieldText
+            data-test-subj="user-name"
```

Comment:
Just curious whether this is equivalent as "id"

Reply:
`data-test-subj` is attribute to tag your test subject. We can say that `data-test-subj` is equivalent to `id`, but it used only for testing purpose. And I observed that upstream is using data-test-subj for testing. 
Ref: https://www.elastic.co/guide/en/kibana/master/development-tests.html